extensions/wikia/ContentFeeds/ContentFeeds.class.php

```diff
@@ -219,13 +219,13 @@ public static function dummyRecentImagesParserHook( $input, $args, $parser ) {
 		return "Życie..";
 	}
 
-	public static function specialNewImagesHook( $images, $gallery, $limit ) {
```

Comment:
Is this the custom limit you mentioned in the PR description that users don't seem to use much?  Maybe in future PRs make a note of things like this as in-line annotations so reviewers know they're looking at the right thing.